libs/dh/profile/feature-profile-modal/src/lib/dh-profile-modal/dh-profile-modal.service.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DhProfileModalService {
+  private profileUpdateSubject = new Subject<void>();
+
+  public onProfileUpdate$ = this.profileUpdateSubject.asObservable();
```

Comment:
Question: Isnt a subject already a observable?

Reply:
Yes, it is. The `asObservable()` pattern is for protecting the subject, so its `next` is only called via the method below.